.github/workflows/build_documentation.yml

```diff
@@ -14,6 +14,6 @@ jobs:
       package: course
       path_to_docs: course/chapters/
       additional_args: --not_python_module
-      languages: en es fr pt th tr
+      languages: en es fr he pt th tr
```

Comment:
FYI I've added Hebrew to the list of languages here to test we can generate the website :)